android/src/main/java/com/henninghall/date_picker/DatePickerManager.java

```diff
@@ -77,11 +81,24 @@ public void setMinuteInterval(PickerView view, @Nullable int interval) throws Ex
     view.setMinuteInterval(interval);
   }
 
+  @ReactProp(name = "utc")
+  public void setUtc(PickerView view, @Nullable boolean utc) throws Exception {
+    this.utc = utc;
+  }
+
+
+
   @Override
   protected void onAfterUpdateTransaction(PickerView view) {
     super.onAfterUpdateTransaction(view);
-      view.updateDisplayValuesIfNeeded();
-      view.setDate(Utils.isoToDate(date));
+
+    TimeZone timeZone = utc ? TimeZone.getTimeZone("UTC") : TimeZone.getDefault();
+
+    view.updateDisplayValuesIfNeeded();
```

Comment:
view.updateDisplayValuesIfNeeded() should probably be the last thing that happens in this function

Reply:
updateDisplayValuesIfNeeded just update which options are available, it looks like it should be the first thing that runs, otherwise the first setDate will fail because there are no options to select.

Reply:
Sorry my bad. Yes setDate must be the last thing that happens but all the other settings coming from props (max/min date etc) must be set before updateDisplayValuesIfNeeded. Fixed it in 633d237!